[2023-12-17T11:19:11.142+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-17T11:19:11.148+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table'
[2023-12-17T11:19:11.259+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-17T11:19:11.260+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table'
[2023-12-17T11:19:11.341+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-08T04:00:00+00:00 [queued]>
[2023-12-17T11:19:11.376+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-08T04:00:00+00:00 [queued]>
[2023-12-17T11:19:11.377+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-17T11:19:11.552+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Create_Table> on 2023-12-08 04:00:00+00:00
[2023-12-17T11:19:11.574+0000] {standard_task_runner.py:57} INFO - Started process 2449 to run task
[2023-12-17T11:19:11.583+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'air_quality', 'Create_Table', 'scheduled__2023-12-08T04:00:00+00:00', '--job-id', '408', '--raw', '--subdir', 'DAGS_FOLDER/airQuality.py', '--cfg-path', '/tmp/tmphipkuq30']
[2023-12-17T11:19:11.599+0000] {standard_task_runner.py:85} INFO - Job 408: Subtask Create_Table
[2023-12-17T11:19:11.782+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-17T11:19:11.784+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table'
[2023-12-17T11:19:11.792+0000] {task_command.py:416} INFO - Running <TaskInstance: air_quality.Create_Table scheduled__2023-12-08T04:00:00+00:00 [running]> on host 1a58977f5573
[2023-12-17T11:19:12.011+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ghifari' AIRFLOW_CTX_DAG_ID='air_quality' AIRFLOW_CTX_TASK_ID='Create_Table' AIRFLOW_CTX_EXECUTION_DATE='2023-12-08T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-08T04:00:00+00:00'
[2023-12-17T11:19:12.042+0000] {base.py:73} INFO - Using connection ID 'neonServerDB-Postgres' for task execution.
[2023-12-17T11:19:13.201+0000] {airQuality.py:160} INFO - Table air_quality Already Exist
[2023-12-17T11:19:13.267+0000] {python.py:194} INFO - Done. Returned value was: 
        CREATE TABLE air_quality (
            id serial primary key, 
            city varchar(250),
            country varchar(250), 
            unit varchar(20), 
            last_value float8, 
            lat float8,
            lng float8
        )
    
[2023-12-17T11:19:13.317+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=air_quality, task_id=Create_Table, execution_date=20231208T040000, start_date=20231217T111911, end_date=20231217T111913
[2023-12-17T11:19:13.397+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-17T11:19:13.460+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-12-17T12:18:19.234+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-17T12:18:19.240+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table'
[2023-12-17T12:18:19.348+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-17T12:18:19.354+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table'
[2023-12-17T12:18:19.442+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-08T04:00:00+00:00 [queued]>
[2023-12-17T12:18:19.479+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-08T04:00:00+00:00 [queued]>
[2023-12-17T12:18:19.485+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-17T12:18:19.552+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Create_Table> on 2023-12-08 04:00:00+00:00
[2023-12-17T12:18:19.559+0000] {standard_task_runner.py:57} INFO - Started process 4151 to run task
[2023-12-17T12:18:19.587+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'air_quality', 'Create_Table', 'scheduled__2023-12-08T04:00:00+00:00', '--job-id', '618', '--raw', '--subdir', 'DAGS_FOLDER/airQuality.py', '--cfg-path', '/tmp/tmpw2bp0jfj']
[2023-12-17T12:18:19.592+0000] {standard_task_runner.py:85} INFO - Job 618: Subtask Create_Table
[2023-12-17T12:18:19.771+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-17T12:18:19.772+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table'
[2023-12-17T12:18:19.777+0000] {task_command.py:416} INFO - Running <TaskInstance: air_quality.Create_Table scheduled__2023-12-08T04:00:00+00:00 [running]> on host 1a58977f5573
[2023-12-17T12:18:20.196+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ghifari' AIRFLOW_CTX_DAG_ID='air_quality' AIRFLOW_CTX_TASK_ID='Create_Table' AIRFLOW_CTX_EXECUTION_DATE='2023-12-08T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-08T04:00:00+00:00'
[2023-12-17T12:18:20.255+0000] {base.py:73} INFO - Using connection ID 'neonServerDB-Postgres' for task execution.
[2023-12-17T12:18:20.813+0000] {airQuality.py:161} INFO - Table air_quality Already Exist
[2023-12-17T12:18:20.860+0000] {python.py:194} INFO - Done. Returned value was: 
        CREATE TABLE air_quality (
            id serial primary key, 
            city varchar(250),
            country varchar(250), 
            unit varchar(20), 
            last_value float8, 
            lat float8,
            lng float8
        )
    
[2023-12-17T12:18:20.933+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=air_quality, task_id=Create_Table, execution_date=20231208T040000, start_date=20231217T121819, end_date=20231217T121820
[2023-12-17T12:18:21.137+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-17T12:18:21.242+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-12-17T12:18:53.569+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-17T12:18:53.571+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table'
[2023-12-17T12:18:53.632+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-17T12:18:53.633+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table'
[2023-12-17T12:18:53.706+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-08T04:00:00+00:00 [queued]>
[2023-12-17T12:18:53.729+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-08T04:00:00+00:00 [queued]>
[2023-12-17T12:18:53.730+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-17T12:18:53.763+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Create_Table> on 2023-12-08 04:00:00+00:00
[2023-12-17T12:18:53.773+0000] {standard_task_runner.py:57} INFO - Started process 4472 to run task
[2023-12-17T12:18:53.782+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'air_quality', 'Create_Table', 'scheduled__2023-12-08T04:00:00+00:00', '--job-id', '685', '--raw', '--subdir', 'DAGS_FOLDER/airQuality.py', '--cfg-path', '/tmp/tmpflopji5x']
[2023-12-17T12:18:53.787+0000] {standard_task_runner.py:85} INFO - Job 685: Subtask Create_Table
[2023-12-17T12:18:53.901+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-17T12:18:53.903+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table'
[2023-12-17T12:18:53.907+0000] {task_command.py:416} INFO - Running <TaskInstance: air_quality.Create_Table scheduled__2023-12-08T04:00:00+00:00 [running]> on host 1a58977f5573
[2023-12-17T12:18:54.072+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ghifari' AIRFLOW_CTX_DAG_ID='air_quality' AIRFLOW_CTX_TASK_ID='Create_Table' AIRFLOW_CTX_EXECUTION_DATE='2023-12-08T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-08T04:00:00+00:00'
[2023-12-17T12:18:54.094+0000] {base.py:73} INFO - Using connection ID 'neonServerDB-Postgres' for task execution.
[2023-12-17T12:18:54.693+0000] {airQuality.py:161} INFO - Table air_quality Already Exist
[2023-12-17T12:18:54.715+0000] {python.py:194} INFO - Done. Returned value was: 
        CREATE TABLE air_quality (
            id serial primary key, 
            city varchar(250),
            country varchar(250), 
            unit varchar(20), 
            last_value float8, 
            lat float8,
            lng float8
        )
    
[2023-12-17T12:18:54.750+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=air_quality, task_id=Create_Table, execution_date=20231208T040000, start_date=20231217T121853, end_date=20231217T121854
[2023-12-17T12:18:54.802+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-17T12:18:54.850+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-12-28T16:51:54.253+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:51:54.266+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:51:54.465+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:51:54.478+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:51:54.681+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-08T04:00:00+00:00 [queued]>
[2023-12-28T16:51:54.713+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-08T04:00:00+00:00 [queued]>
[2023-12-28T16:51:54.718+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-28T16:51:54.776+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Create_Table> on 2023-12-08 04:00:00+00:00
[2023-12-28T16:51:54.789+0000] {standard_task_runner.py:57} INFO - Started process 256 to run task
[2023-12-28T16:51:54.821+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'air_quality', 'Create_Table', 'scheduled__2023-12-08T04:00:00+00:00', '--job-id', '874', '--raw', '--subdir', 'DAGS_FOLDER/airQuality.py', '--cfg-path', '/tmp/tmpvyqfajt8']
[2023-12-28T16:51:54.835+0000] {standard_task_runner.py:85} INFO - Job 874: Subtask Create_Table
[2023-12-28T16:51:55.218+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:51:55.220+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:51:55.233+0000] {task_command.py:416} INFO - Running <TaskInstance: air_quality.Create_Table scheduled__2023-12-08T04:00:00+00:00 [running]> on host d67dd4682ccf
[2023-12-28T16:51:55.934+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ghifari' AIRFLOW_CTX_DAG_ID='air_quality' AIRFLOW_CTX_TASK_ID='Create_Table' AIRFLOW_CTX_EXECUTION_DATE='2023-12-08T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-08T04:00:00+00:00'
[2023-12-28T16:51:55.971+0000] {base.py:73} INFO - Using connection ID 'neonServerDB-Postgres' for task execution.
[2023-12-28T16:52:00.261+0000] {airQuality.py:139} INFO - Table air_quality Already Exist
[2023-12-28T16:52:00.370+0000] {python.py:194} INFO - Done. Returned value was: 
        CREATE TABLE air_quality (
            id serial primary key, 
            city varchar(250),
            country varchar(250), 
            unit varchar(20), 
            last_value float8, 
            lat float8,
            lng float8
        )
    
[2023-12-28T16:52:00.462+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=air_quality, task_id=Create_Table, execution_date=20231208T040000, start_date=20231228T165154, end_date=20231228T165200
[2023-12-28T16:52:00.598+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-28T16:52:00.686+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-12-28T16:54:58.168+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:54:58.171+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:54:58.397+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:54:58.400+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:54:58.708+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-08T04:00:00+00:00 [queued]>
[2023-12-28T16:54:58.776+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: air_quality.Create_Table scheduled__2023-12-08T04:00:00+00:00 [queued]>
[2023-12-28T16:54:58.792+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-28T16:54:58.865+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): Create_Table> on 2023-12-08 04:00:00+00:00
[2023-12-28T16:54:58.884+0000] {standard_task_runner.py:57} INFO - Started process 1255 to run task
[2023-12-28T16:54:58.899+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'air_quality', 'Create_Table', 'scheduled__2023-12-08T04:00:00+00:00', '--job-id', '1127', '--raw', '--subdir', 'DAGS_FOLDER/airQuality.py', '--cfg-path', '/tmp/tmp7cms8bau']
[2023-12-28T16:54:58.913+0000] {standard_task_runner.py:85} INFO - Job 1127: Subtask Create_Table
[2023-12-28T16:54:59.238+0000] {logging_mixin.py:154} INFO - Changing /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table permission to 509
[2023-12-28T16:54:59.241+0000] {logging_mixin.py:154} INFO - Failed to change /opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table permission to 509: [Errno 1] Operation not permitted: '/opt/***/logs/dag_id=air_quality/run_id=scheduled__2023-12-08T04:00:00+00:00/task_id=Create_Table'
[2023-12-28T16:54:59.252+0000] {task_command.py:416} INFO - Running <TaskInstance: air_quality.Create_Table scheduled__2023-12-08T04:00:00+00:00 [running]> on host d67dd4682ccf
[2023-12-28T16:54:59.791+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ghifari' AIRFLOW_CTX_DAG_ID='air_quality' AIRFLOW_CTX_TASK_ID='Create_Table' AIRFLOW_CTX_EXECUTION_DATE='2023-12-08T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-08T04:00:00+00:00'
[2023-12-28T16:54:59.834+0000] {base.py:73} INFO - Using connection ID 'neonServerDB-Postgres' for task execution.
[2023-12-28T16:55:04.099+0000] {airQuality.py:139} INFO - Table air_quality Already Exist
[2023-12-28T16:55:04.174+0000] {python.py:194} INFO - Done. Returned value was: 
        CREATE TABLE air_quality (
            id serial primary key, 
            city varchar(250),
            country varchar(250), 
            unit varchar(20), 
            last_value float8, 
            lat float8,
            lng float8
        )
    
[2023-12-28T16:55:04.251+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=air_quality, task_id=Create_Table, execution_date=20231208T040000, start_date=20231228T165458, end_date=20231228T165504
[2023-12-28T16:55:04.664+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-28T16:55:04.788+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
